packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -70,9 +426,15 @@ class WidgetInspector extends StatefulWidget {
 class _WidgetInspectorState extends State<WidgetInspector>
     with WidgetsBindingObserver {
 
+  _WidgetInspectorState() : selection = WidgetInspectorService.instance.selection {
+    WidgetInspectorService.instance.selectionChangedCallback = () {
+      setState(() {});
```

Comment:
all setState methods should either actually set some state, or have a comment inside the closure saying what it is that changed (which the build method depends on)

Reply:
Done. I felt bad about that setState call.